server/utils/checkYourAnswersUtils.ts

```diff
@@ -36,23 +36,26 @@ export const getTaskAnswersAsSummaryListItems = (
 ): Array<SummaryListItem | QuestionAndAnswer> => {
   const items: Array<SummaryListItem | QuestionAndAnswer> = []
 
+  // Get the latest question schema
```

Comment:
I try not to add comments but this code felt like variable and method naming alone didn't provide enough.